frontend/console/src/features/modules/decls/ConfigPanel.tsx

```diff
@@ -0,0 +1,18 @@
+import type { Config } from '../../../protos/xyz/block/ftl/v1/schema/schema_pb'
+import { PanelHeader } from './PanelHeader'
+import { TypeEl } from './TypeEl'
+
+export const ConfigPanel = ({ value, moduleName, declName }: { value: Config; moduleName: string; declName: string }) => {
+  return (
+    <div className='py-2 px-4'>
+      <PanelHeader exported={false} comments={value.comments}>
+        <p>
+          Config: {moduleName}.{declName}
+        </p>
```

Comment:
It seems like all the `PanelHeader` have this `<p>` tag in them, should we add that to the header itself, or keep separate? Or should we remove here and just show whatever text?

Reply:
I'll just pull out all the <p> tags since we they don't actually have an effect. I think we needed that in there at some point early on and now it's just not necessary.